simcon_project/users/views/view_response.py

```diff
@@ -12,9 +12,14 @@
 @user_passes_test(is_authenticated)
 def view_response(request, pk):
     response = get_object_or_404(TemplateResponse, pk=pk)
-
+    if request.method == "POST":
+        response_table = TemplateResponse.objects.all()
+        items_to_delete = request.POST.getlist('delete_items')
+        response_table.filter(pk__in=items_to_delete).delete()
+        return HttpResponseRedirect(reverse('researcher-view'))
     nodes = []
-    num_nodes = TemplateNodeResponse.objects.filter(parent_template_response=response).count()
+    num_nodes = TemplateNodeResponse.objects.filter(
+        parent_template_response=response).count()
```

Comment:
This isn't your fault but line 55 throws an error when you try to submit a new overall feedback. Seems to work if it's changed to this:
return HttpResponseRedirect(reverse('view-response', kwargs={'pk': pk}))

Reply:
ill fix this once im off of work! im confused because i didnt touch that...